### ISSUE TITLE
Applying the fix doesn't always update instances of Module.Constructor

### ISSUE THREAD

**Issue by wolfadex** [sparksp/elm-review-imports#40]
Given two modules
```Elm
module Main exposing (main)

import Example
import Html


main : Html.Html msg
main =
    Html.text <|
        case Just Example.Foo of
            Just Example.Foo ->
                "just foo"

            Just Example.Bar ->
                "just bar"

            Nothing ->
                "nothing"
```
and
```Elm
module Example exposing (FooBar(..))


type FooBar
    = Foo
    | Bar
```
and an elm-review config of
```Elm
module ReviewConfig exposing (config)

import NoInconsistentAliases
import Review.Rule exposing (Rule)


config : List Rule
config =
    [ NoInconsistentAliases.config
        [ ( "Example", "E" ) ]
        |> NoInconsistentAliases.noMissingAliases
        |> NoInconsistentAliases.rule
    ]

```
applying the fix converts `Main.elm` to
```Elm
module Main exposing (main)

import Example as E
import Html


main : Html.Html msg
main =
    Html.text <|
        case Just E.Foo of
            Just Example.Foo ->
                "foo"

            Just Example.Bar ->
                "bar"

            Nothing ->
                "nothing"
```

## Expected
`Just Example.Foo` to be changed to `Just E.Foo`

## Actual
`Just Example.Foo` isn't modified

**Comment by sparksp** [comment]
Thanks for the very detailed bug report! I'll take a look at this later hopefully. 👍

**Comment by sparksp** [comment]
This has been published in 1.0.1.

**Comment by wolfadex** [comment]
That was impressively fast, thank you!
